flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/GroupAggregateTestPrograms.java

```diff
@@ -252,6 +255,15 @@ public class GroupAggregateTestPrograms {
                                     + "my_avg(e, a) as s1, "
                                     + "my_concat(d) as c1 "
                                     + "FROM source_t GROUP BY e")
+                    .runTableApi(
```

Comment:
so far all ExecNode tests are SQL based, can we remove this. otherwise it encourages people to mix table api and sql. we can also copy those 4 tests over to `QueryOperationTestPrograms`.